budget-binder-multiplatform-app/src/commonMain/kotlin/de/hsfl/budgetBinder/domain/usecase/AuthUseCase.kt

```diff
@@ -18,17 +19,17 @@ class RegisterUseCase(private val repository: AuthRepository) {
                     emit(DataResponse.Success(it))
                 } ?: response.error?.let { error ->
```

Comment:
Mir ist aufgefallen, dass in allen Use Cases response.error?.let steht, es sollte aber response.error!!.let sein.
wenn, aus welchen gründen auch immer, falsche Antworten vom Server kommen ,die nichts von beiden mitschicken. würde der Satte für immer in Loading feststecken. Mit der Änderung würde die Anfrage jedenfalls mit einem Fehler fehlschlagen und nicht stuck sein.